_episodes/03-dplyr.md

```diff
@@ -223,559 +223,439 @@ You can also nest functions (i.e. one function inside of another), like this:
-##  6 F     OL               31.1 
-##  7 F     OT               24.8 
-##  8 F     OX               21   
-##  9 F     PB               30.2 
-## 10 F     PE               22.8 
-## 11 F     PF                7.97
-## 12 F     PH               30.8 
-## 13 F     PL               19.3 
-## 14 F     PM               22.1 
-## 15 F     PP               17.2 
-## # … with 49 more rows
+## Error in print.default(n = 15): argument "x" is missing, with no default
 ```
 
-Once the data are grouped, you can also summarize multiple variables at the same
-time (and not necessarily on the same variable). For instance, we could add a
-column indicating the minimum weight for each species for each sex:
+Once the data are grouped, you can also summarize multiple variables at the same time (and not necessarily on the same variable). For instance, we could add a column indicating the minimum weight for each species for each sex:
 
 
 ```r
-surveys %>%
-  filter(!is.na(weight)) %>%
-  group_by(sex, species_id) %>%
-  summarize(mean_weight = mean(weight),
-            min_weight = min(weight))
+plant_phys %>%
+  group_by(Year, Species) %>%
+  summarize(mean_photo = mean(Photo),
+            min_photo = min(Photo))
 ```
 
 ```
-## # A tibble: 64 x 4
-## # Groups:   sex [3]
-##    sex   species_id mean_weight min_weight
-##    <chr> <chr>            <dbl>      <int>
-##  1 F     BA                9.16          6
-##  2 F     DM               41.6          10
-##  3 F     DO               48.5          12
-##  4 F     DS              118.           45
-##  5 F     NL              154.           32
-##  6 F     OL               31.1          10
-##  7 F     OT               24.8           5
-##  8 F     OX               21            20
-##  9 F     PB               30.2          12
-## 10 F     PE               22.8          11
-## # … with 54 more rows
+## # A tibble: 16 x 4
+## # Groups:   Year [2]
+##     Year Species mean_photo min_photo
+##    <int> <chr>        <dbl>     <dbl>
+##  1  2014 AC            12.0      6.60
+##  2  2014 AG            18.8     10.3 
+##  3  2014 CD            11.4      6.71
+##  4  2014 PV            19.3      6.56
+##  5  2014 RG            15.8      2.47
+##  6  2014 SC            11.0      1.82
+##  7  2014 SN            19.3      9.78
+##  8  2014 VB            10.7      3.13
+##  9  2015 AC            12.7      5.14
+## 10  2015 AG            18.1      8.24
+## 11  2015 CD            13.3      7.38
+## 12  2015 PV            24.7     10.6 
+## 13  2015 RG            17.4      8.35
+## 14  2015 SC            19.8      6.58
+## 15  2015 SN            29.5     11.7 
+## 16  2015 VB            12.2      1.08
 ```
 
-It is sometimes useful to rearrange the result of a query to inspect the values. For instance, we can sort on `min_weight` to put the lighter species first:
+It is sometimes useful to rearrange the result of a query to inspect the values. For instance, we can sort on `min_photo` to put the less photosynthetic first:
```

Comment:
Could we continue the theme from before and sort on Cond_ratio instead of min_photo?